lib/src/sensor_device.h

```diff
@@ -109,8 +110,8 @@ class Sensor
     std::string topicSuffix() const;
 
 protected:
-    const AssetState::Asset *_asset{nullptr};
-    const AssetState::Asset *_parent{nullptr};
+    std::shared_ptr<AssetState::Asset> _asset{nullptr};
```

Comment:
what is the reason to change ptr w/ smartPtr?

Reply:
More secure